server.py

```diff
@@ -1,90 +1,385 @@
 #!/eecs/research/asr/mingbin/python-workspace/hopeless/bin/python
+# -*- coding: utf-8 -*-
 
-from flask import Flask, render_template,request, json, jsonify
-from subprocess import Popen, PIPE, call
+from flask import Flask, render_template, request, jsonify
+from subprocess import Popen, PIPE
+import os, time, sys, argparse, logging, pandas
 from pandas import DataFrame
-from nltk.tokenize import sent_tokenize, word_tokenize
 from fofe_ner_wrapper import fofe_ner_wrapper
+from langdetect import detect
+from pycorenlp import StanfordCoreNLP
+import pprint
 
-import argparse, logging, pandas
-logger = logging.getLogger( __name__ )
+reload(sys)
+sys.setdefaultencoding("utf-8")
 
+logger = logging.getLogger(__name__)
 
-cls2ner = [ 'PER', 'LOC', 'ORG', 'MISC' ]
-app = Flask( __name__ )
+cls2ner = ['PER', 'LOC', 'ORG', 'MISC']
+app = Flask(__name__)
 
+PATH = "/local/scratch/nana/EL-GUI/ELdata/file_gen/"
+FILENAME = "ENG_NW_0000"
+FILE_EXT = ".nw.xml"
+FILE_LIST = "/local/scratch/nana/EL-GUI/ELdata/src/eng.all.list"
+INFO_PATH = "/local/scratch/nana/EL-GUI/ELdata/src/test.el.iflytek.eng.tab"
+OUTPUT_PATH = "/local/scratch/nana/EL-GUI/ELdata/rst/edl.iNCML.tsv.name"
+CANDIDATE_FILE = "/local/scratch/nana/EL-GUI/ELdata/rst/out.candidate.txt"
```

Comment:
Please refer to these two scripts and see how I set up the paths:
https://github.com/xmb-cipher/torch-fofe-lm/blob/master/path.sh
https://github.com/xmb-cipher/torch-fofe-lm/blob/master/run.sh

Except that the root directory is hard-coded, every path is relative. 
You will see how it set up the environment in "path.sh". It is a bad practice to assume your collaborators are able to reproduce your environment, **but that's my fault**. I will fix it after I review everything.